src/common/collection_column_specs/genome_attribs-PMI.yml

```diff
@@ -19,6 +19,10 @@ columns:
      type: string
      filter_strategy: identity
 
+   - key: kbase_display_name
+     type: string
+     filter_strategy: identity
```

Comment:
ngram would probably be better here so users can search for object names they don't quite remember

Reply:
👍